mlrun/api/main.py

```diff
@@ -312,19 +318,44 @@ async def _start_log_and_update_runs(
     if not runs:
         return
 
-    # each result contains either run_uid or None
-    # if it's None it means something went wrong, and we should skip it
-    # if it's run_uid it means we requested logs collection for it and we should update it's requested_logs field
-    results = await asyncio.gather(
-        *[
+    # the max number of consecutive start log requests for a run before we mark it as requested logs collection
+    # basically represents the grace period before the run's resources are deleted
+    max_consecutive_start_log_requests = int(
```

Comment:
IIUC by default its 1440 which seems a lot to me.
I understand the incentive that only after 4 hours we can be sure that the resource was deleted but even after 1 hour we can safely say that the pod is not coming back. I think it should be configurable and default to 1 hour.